doc/data/messages/c/confusing-with-statement/good.py

```diff
@@ -1 +1,5 @@
-# This is a placeholder for correct code for this message.
+from contextlib import ExitStack
+
+with ExitStack() as stack:
+    fh1 = stack.enter_context(open('file1.txt', 'w'))
+    fh2 = stack.enter_context(open('file2.txt', 'w'))
```

Comment:
How about this:
```suggestion
with open('file.txt', 'w') as fh1:
    with open('file.txt', 'w') as fh2:
        pass
```
Using ExitStack is probably worse for performance because it has multiple unavoidable call, it add an import and deviate from the standard way of doing thing (you have to search for what ``ExitStack`` and ``enter_context`` do)

Reply:
😆 
```
doc/data/messages/c/confusing-with-statement/good.py:1:5: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
doc/data/messages/c/confusing-with-statement/good.py:2:9: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
```

but I'll pick another example

Reply:
Shouldn't this message be disabled ? We can specify the encoding too.